src/components/my-history/custom-my-history.vue

```diff
@@ -16,12 +16,28 @@
             @click="closeModal"
             data-test="my-history-query"
             :suggestion="suggestion"
-            suggestionClass="x-suggestion"
+            suggestionClass="x-suggestion x-w-[320px]"
           >
             <HistoryIcon class="max-desktop:x-icon-lg" />
 
             <div class="x-flex x-flex-col x-gap-2">
-              <p>{{ suggestion.query }}</p>
+              <p class="hover:x-underline">{{ suggestion.query }}</p>
+              <div
+                v-if="suggestion.selectedFilters && suggestion.selectedFilters.length > 0"
+                class="x-text1-sm x-flex x-text-lead-50"
+              >
+                <p class="x-line-clamp-1">
+                  <span
+                    v-for="filter in suggestion.selectedFilters"
+                    :key="filter.id"
+                    class="x-pr-8"
+                  >
+                    {{ label(filter) }}
+                  </span>
+                </p>
+                <p class="x-pl-8">({{ suggestion.selectedFilters.length }})</p>
```

Comment:
You're adding extra `8px` between the last filter and the filter count with this padding.
Also this should be an span

```suggestion
                <span>({{ suggestion.selectedFilters.length }})</span>
```